designs/Net5PlatformAndPlatformVersion.md

```diff
@@ -0,0 +1,177 @@
+
+## class CompatibilityTests
+
+Source: test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityTests.cs
+
+`[Theory] Compatibility_FrameworksAreCompatible` data needs to be modified such that Platforms are supported, and all expected platforms are included in the tests.
+
+## class NuGetFrameworkParseTests
+
+Source: test/NuGet.Core.Tests/NuGet.Frameworks.Test/NuGetFrameworkParseTests.cs
+
+`[Theory] NuGetFramework_Basic` needs additional test cases to make sure Platform and PlatformVersion are being parsed correctly from the "long" tfm format.
+
+## class NuGetFrameworkTests
+
+Source: test/NuGet.Core.Tests/NuGet.Frameworks.Test/NuGetFrameworkTests.cs
+
+An additional test should be added to make sure things like `net50-android10` parse to `net5.0-android10.0`.
+
+## Remaining work/questions
+
+1. Ensure Pack target passes TargetPlatformVersion to .nuspec generator. (Use `0.0` as a sentinel)
+1. Fail parsing/creation of `NuGetFramework` when an unknown OS flavor is being used -- slam an Unsupported tfi into it??
+
+## Out of scope for this
+
+1. We need to make sure there's some kind of information/warning available when `net50` is used, since [we're going to warn now if the `.` is missing, starting from `net5.0`](https://github.com/NuGet/Home/issues/9215). I'm not sure what level to do this at.
+1. Design NuGet manifest entries for TargetPlatformVersion and TargetPlatformMinVersion (not sure what "design" is needed -- TPV should be treated as part of the tfm itself. TargetPlatformMinVersion is a separate, new attribute. This is a bit more involved than just passing it through.)
+1. Ensure Pack target passes TargetPlatformMinVersion to .nuspec generator.
+1. Fail pack when packing equivalent TFMs like net5.0-ios and net5.0-ios13.0 (note: we do want to eventually allow multi-targeting with equivalent TFMs, but that's out of scope for this work, and [seems to be a Big Task](https://github.com/NuGet/Home/issues/5154) -- also, we don't support them right now, so this isn't an issue yet)
+1. Produce an error when a OS flavored .NET 5 TFM is consumed but the package has no TPV in either folder name of the manifest. Is this supposed to be during restore??
+1. Produce a warning when a project consumes a library with a higher `TargetPlatformMinVersion` (related to 5154)
+1. When present, msbuild props like `TargetPlatformIdentifier` and `TargetPlatformVersion` should supersede the corresponding data in `NuGetFramework` objects. (this is related to 5154, too)
+
+## Follow-up
+
+1. Compatibility tables need to be extended to include compatibility for all the Xamarin TFMs to appropriate `net5.0-xxxx1.2` TFMs. I'm not sure we do anything similar right now, so this might actually be new functionality? -- send an email to Immo because our understanding is that we're not actually doing this. cc nikolche/rob

```

Comment:
My memory from the meetings were that old Xamarin TFMs were going to be incompatible, and target fallback could be used if desired. But it's good to check.

Reply:
Turns out -- they're gonna be compatible! The current spec is correct and up to date.